dvc/render/image_converter.py

```diff
@@ -45,9 +45,7 @@ def convert(
         if path:
             if not os.path.isdir(path):
                 os.makedirs(path, exist_ok=True)
-            src = self._write_image(
-                os.path.abspath(path), revision, filename, data
```

Comment:
When creating html with images under `dvc_plots/static` its better if we provide relpath. That way we can easily share the produced HTML with other users.